src/styles/_variables.scss

```diff
@@ -68,3 +68,6 @@ $navbar-item-hover-background-color: #e9ecef;
 
 // title
 $title-line-height: 1.2;
+
+// mobile section
+$mobile-section-padding: 1rem;
```

Comment:
The `.section` has been defined in global.scss. You can remove this duplicate definition here.